### DISCUSSION TITLE
Centering Plotly in Rmarkdown

### DISCUSSION THREAD

**Discussion by RanggaGemilang** [teamalgoritma/community#277] [Q&A] [answered]
Halo saya ingin agar plot saya ke tengah / "centered", bagaimana caranya ya? Saya sudah coba dengan fig-align ="center" di chunk saya tetap tidak bisa. 

Untuk plot saya menggunakan **ggplotly/plotly **.
![centerQ](https://user-images.githubusercontent.com/107049922/186337728-44b98ab0-d168-464d-9639-5794b189971e.png)

Mohon bantuannya yaa, terima kasih!

**Comment by NabiilahArdini** [comment, marked as answer]
Halo Bapak, pengaturan HTML pada plotly memang berbeda dengan R Markdown, sehingga perlu dibuatkan pengaturan khusus. Untuk membuat output plotly ter-align center, silahkan gunakan code **<center>** dan **</center>** diantara chunk untuk pembuatan plotly. Berikut contohnya:

```
        <center>
        
        ```{r, echo = FALSE, out.width = "60%"}
        library(dplyr)
        library(ggplot2)
        library(plotly)
        
        plot <- iris %>% 
          ggplot(aes(x = Species, y = Sepal.Width)) +
          geom_boxplot()
        
        ggplotly(plot)
        ```
        
        </center>
```

<img width="634" alt="image" src="https://user-images.githubusercontent.com/51941102/187125363-2d1cc655-450d-444a-9bc7-3a0a93b6f417.png">
